sample-compose/README.md

```diff
@@ -0,0 +1,51 @@
+# Android Compose Sample
+
+A sample project generated from our template using Compose layouts
+
+## Setup
+
+- Clone the project
+- Run the project with Android Studio
+
+## Linter and static code analysis
+
+- Lint:
+
+```
+$ ./gradlew lint
+```
+
+Report is located at: `./app/build/reports/lint/`
+
+- Detekt
+
+```
+$ ./gradlew detekt
+```
+
+Report is located at: `./build/reports/detekt`
+
+## Testing
+
+- Run unit testing:
+
+```
+$ ./gradlew app:testStagingDebugUnitTest
+$ ./gradlew data:testDebugUnitTest
+$ ./gradlew domain:test
+```
+
+- Run unit testing with coverage:
+
+```
+$ ./gradlew jacocoTestReport
+```
+
+Report is located at: `./app/build/reports/jacoco/`
+
+## Build and deploy
+
+For `release` builds, we need to provide release keystore and signing properties:
+
+- Put the `release.keystore` file at root `config` folder.
+- Put keystore signing properties in `signing.properties`
```

Comment:
I have a question on this 🙋🏻 . Do we need to add the `copyright` here too? @Tuubz  @chornerman

Reply:
Hmmm, what do you mean by copyright? 😯

Reply:
@Tuubz This part:
![Screen Shot 2565-08-17 at 4 40 32 PM](https://user-images.githubusercontent.com/12026942/185087358-b2bdf933-a744-48c6-9486-33eb72cb1b5d.png)


Ref: https://github.com/nimblehq/github-action-runner-android

Reply:
Aha, I see 👍 

Yeah, we can add it in a future issue 🚀